packages/shared/lib/core/account/actions/loadAccount.ts

```diff
@@ -6,8 +6,8 @@ import { buildAccountStateAndMetadata } from './buildAccountStateAndMetadata'
 export async function loadAccount(account: IAccount): Promise<IAccountState> {
     // Temporary sync on load until we enable background sync and event listeners
     await account.sync({ syncIncomingTransactions: true })
-    const metadata = getAccountMetadataById(account?.meta?.index.toString())
-    let accountState
+    const metadata = getAccountMetadataById(account?.meta?.index)
```

Comment:
We can and we should!

Reply:
Actually, we cannot, since we're inputting a rust object on this side which doesn't have the same properties as our own IAccountState. We're only building the metadata after we fetched it through the index.